UnityProject/Assets/Light2D/Scripts/CustomSprite.cs

```diff
@@ -96,7 +96,7 @@ protected virtual void OnEnable()
             _mesh.name = GeneratedMeshName;
 
             _tangents = new Vector4[4];
-            for (int i = 0; i < _tangents.Length; i++)
+            for (var i = 0; i < _tangents.Length; i++)
```

Comment:
I do prefer ints and floats to be explicit.

Reply:
yeah, primitives better be explicit

Reply:
Will add it atleast to the codingstyle this way, even if we don't use the automated cleanup.